content/en/blog/2022-06-24-transparently-mocking-grpc-services.md

```diff
@@ -0,0 +1,339 @@
+---
+title: "Transparently Mocking gRPC Services"
+slug: "transparently-mocking-grpc-services"
+date: "2022-06-24T12:30:33-05:00"
+draft: true
+image_webp: images/blog/blue_laser.webp
+image: images/blog/blue_laser.jpg
+author: Patrick Deziel
+description: "gRPC is a common framework used to facilitate communication between microservices, but testing these services can be a challenge. In this post we will present a simple strategy for mocking gRPC services in Go."
+---
+
+gRPC is an effective way of implementing service-to-service APIs. However, there are significantly less tools available for mocking and testing than REST-based services. One option is to setup a live test server, however in this blog post we will demonstrate a more lightweight solution which utilizes a hand-built mock. Don't worry, this is easier than it seems!
```

Comment:
minor suggestions on this line

- instead of `less tools`, use `fewer tools` as the adjective
- instead of `to setup`, use `to set up` since setup is used as a noun rather than a verb
- adding a semi colon and commas, instead of `One option is to setup a live test server, however in this blog`, use `One option is to setup a live test server; However, in this blog post,` since it seems to be two separate clauses.